t/ids_from_pages.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+describe 'Wikidata#ids_from_pages' do
+  around { |test| VCR.use_cassette(titles.first, &test) }
+  subject { WikiData.ids_from_pages('en', titles) }
+
+  describe 'Direct links' do
+    describe 'single title' do
+      let(:titles) { ['Douglas Adams'] }
+      it 'pairs a title with an id' do
+        subject.must_equal('Douglas Adams' => 'Q42')
+      end
+    end
+
+    describe 'multiple titles' do
+      let(:titles) { ['Towel', 'Douglas Adams'] }
+      it 'pairs an id with each title' do
```

Comment:
Naming this one backwards from the other two is a little confusing, as it makes it appear that there's something subtly different going on here…